### PR TITLE
Functional tests - Test minimum purchase total required in order to validate the order

### PR DESCRIPTION
**Pull request by nesrineabdmouleh**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.7.x
| Description?  | Test minimum purchase total required in order to validate the order
| Type?         | refacto
| Category?     | TE
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | no
| How to test?  |  TEST_PATH="functional/BO/13_shopParameters/02_orderSettings/01_orderSettings/general/03_minimumPurchaseTotalRequired.js" URL_FO=yourShopURL npm run specific-test

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/18290)

### CODE REVIEW

**Review by NeOMakinG — DISMISSED** [review]

**Comment by NeOMakinG** [comment]
Travis is red @nesrineabdmouleh

File: tests/puppeteer/campaigns/functional/BO/13_shopParameters/02_orderSettings/01_orderSettings/general/03_minimumPurchaseTotalRequired.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+const ProductPage = require('@pages/FO/product');
+const FOBasePage = require('@pages/FO/FObasePage');
+const HomePage = require('@pages/FO/home');
+const CartPage = require('@pages/FO/cart');
+
+let browser;
+let page;
+const newPurchaseTotalRequired = 100;
+const defaultPurchaseTotalRequired = 0;
+const alertMessage = 'A minimum shopping cart total of €100.00 (tax excl.) is required to validate your order.';
```
Comment (SimonGrn) [inline tests/puppeteer/campaigns/functional/BO/13_shopParameters/02_orderSettings/01_orderSettings/general/03_minimumPurchaseTotalRequired.js]:
```suggestion
const alertMessage = `A minimum shopping cart total of €${newPurchaseTotalRequired}.00 (tax excl.) is required to validate your order.`;
```

**Review by SimonGrn — COMMENTED** [review]
Thanks @nesrineabdmouleh

**Review by NeOMakinG — APPROVED** [review]

File: tests/puppeteer/campaigns/functional/BO/13_shopParameters/02_orderSettings/01_orderSettings/general/03_minimumPurchaseTotalRequired.js (hunk 1 above)
Comment (SimonGrn) [reply tests/puppeteer/campaigns/functional/BO/13_shopParameters/02_orderSettings/01_orderSettings/general/03_minimumPurchaseTotalRequired.js]: You could even format it to take decimals into account ? Just a suggestion

**Review by PierreRambaud — APPROVED** [review]

**Pull request merged**